mungegithub/mungers/inactive-review-handler.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	githubapi "github.com/google/go-github/github"
+	"github.com/spf13/cobra"
+	"k8s.io/test-infra/mungegithub/features"
+	"k8s.io/test-infra/mungegithub/github"
+	"k8s.io/test-infra/mungegithub/mungers/matchers"
+	"time"
+)
+
+const (
+	NOTIFNAME       = "INACTIVE PULL REQUEST"
+	CREATIONTIMECAP = 36 * 30 * 24 * time.Hour //period since PR creation time
+	COMMENTTIMECAP  = 7 * 24 * time.Hour       //period since last IssueComment and PullRequestComment being posted
+	REMINDERNUMCAP  = 5                        //maximum number of times this munger will post reminder IssueComment
+	LEAFOWNERSONLY  = false                    //setting for Blunderbuss to fetch only leaf owners or all owners
+)
+
+type InactiveReviewHandler struct {
+	features *features.Features
+}
+
+func init() {
+	h := &InactiveReviewHandler{}
+	RegisterMungerOrDie(h)
+}
+
+func (i *InactiveReviewHandler) Name() string { return "inactive-review-handler" }
+
+func (i *InactiveReviewHandler) RequiredFeatures() []string {
+	return []string{features.RepoFeatureName, features.AliasesFeature}
+}
+
+func (i *InactiveReviewHandler) Initialize(config *github.Config, features *features.Features) error {
+	i.features = features
+	return nil
+}
+
+// EachLoop is called at the start of every munge loop
+func (i *InactiveReviewHandler) EachLoop() error { return nil }
+
+// AddFlags will add any request flags to the cobra `cmd`
+func (i *InactiveReviewHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
+
+func (i *InactiveReviewHandler) getLatestNonAuthorHumanTime(issueCreatedAt *time.Time, authorName *string, comments []*githubapi.IssueComment, reviewComments []*githubapi.PullRequestComment) *time.Time {
+	return matchers.Items{}.
```

Comment:
We are actually more interested in the count of these filtered items, rather than the time of the last item.

The goal was to trigger on pull-requests that have NEVER commented by anyone but the author/bot.